EIC-EA/EIC-community-D8
# Feature/eicnet 585 group persistant urls

Testing:

- Create Group A and add group purl 'group-a'
- Create Group B and add group purl 'group-b'
- Add a discussion to Group A
- Add a discussion to Group B
- Create a user and join Group A
- Create a user and join Group B
- Re-index everything if needed
- Visit **/group-a/discussions** and **/group-a/members**
- Visit **/group-b/discussions** and **/group-b/members**

You should see different discussions and members per group
